src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,69 @@
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../utils/searchHelper';
+
+enum FilterType {
+  female = 'f',
+  male = 'm',
+}
```

Comment:
Enum's elements should be in CamelCase or in UpperCase 
```suggestion
enum FilterType {
  Female = 'f',
  Male = 'm',
}
```